mapper/s3.go

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mapper
+
+import (
+	"github.com/ernestio/aws-definition-mapper/definition"
+	"github.com/ernestio/aws-definition-mapper/output"
+)
+
+// MapS3Buckets : Maps the s3 buckets from a given input payload.
+func MapS3Buckets(d definition.Definition) []output.S3 {
```

Comment:
This method is not being called anywhere, should be called [here](https://github.com/ernestio/aws-definition-mapper/blob/189f60ac27ba6f9fb4efbfc6128ed836510489d4/mapper/mapper.go#L40)